src/libYARP_OS/src/Os.cpp

```diff
@@ -142,3 +142,22 @@ int yarp::os::fork()
     pid_t pid = yarp::os::impl::fork();
     return pid;
 }
+
+
+#if defined(__APPLE__)
+#include "yarp/os/impl/MacOSAPI.h"
+#endif
+
+
+void yarp::os::setEnergySavingModeEnabled(bool enabled)
+{
+#if defined(__APPLE__)
+    static void* handle = 0;
+    if (!enabled && !handle) {
```

Comment:
If we pass `false` here we should set energy saving to `OFF`, am I right?
But if `enable == false` it seems that we are asking to activate AppNap.

Reply:
We should just "invert" the names of the inner methods. I'll do it

Reply:
Is this done in the latest patch? @claudiofantacci when we are happy with the changes, I think we can merge this...